.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,22 +1,37 @@
+## Overview
 <!--
-  - Thanks for submitting code to Oppia! Please fill out the following as part of
-  - your pull request so we can review your code more easily.
-  -->
+READ ME FIRST:
+Please answer *both* questions below and check off every point from the Essential Checklist!
+If there is no corresponding issue number, fill in N/A where it says [fill_in_number_here] below in point 1.
+-->
 
-## Explanation
-<!--
-  - Explain what your PR does. If this PR fixes an existing bug, please include
-  - "Fixes #bugnum:" in the explanation so that GitHub can auto-close the issue
-  - when this PR is merged.
-  -->
+1. This PR fixes or fixes part of #[fill_in_number_here].
+2. This PR does the following: [Explain here what your PR does and why]
 
-## Checklist
+## Essential Checklist
 <!-- Please tick the relevant boxes by putting an "x" in them. -->
-- [ ] The PR title starts with "Fix #bugnum: ", followed by a short, clear summary of the changes. (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
-- [ ] The PR explanation includes the words "Fixes #bugnum: ..." (or "Fixes part of #bugnum" if the PR only partially fixes an issue).
+- [ ] The PR title and explanation starts with "Fix #bugnum: " (If this PR fixes part of an issue, prefix the title with "Fix part of #bugnum: ...".)
+- [ ] Any changes to [scripts/assets](https://github.com/oppia/oppia-android/tree/develop/scripts/assets) files have rationale included in the PR description.
 - [ ] The PR follows the [style guide](https://github.com/oppia/oppia-android/wiki/Coding-style-guide).
-- [ ] The PR does not contain any unnecessary auto-generated code from Android Studio.
-- [ ] The PR is made from a branch that's **not** called "develop".
-- [ ] The PR is made from a branch that is up-to-date with "develop".
-- [ ] The PR's branch is based on "develop" and not on any other branch.
-- [ ] The PR is **assigned** to an appropriate reviewer in both the **Assignees** and the **Reviewers** sections.
+- [ ] The PR does not contain any unnecessary code changes from Android Studio.
+- [ ] The PR is made from a branch that's **not** called "develop" and is up-to-date with "develop".
+- [ ] The PR is **assigned** to appropriate reviewers. [Reference](https://github.com/oppia/oppia-android/wiki/Guidance-on-submitting-a-PR#clarification-regarding-assignees-and-reviewers-section)
+
+## Proof that changes are correct
+
+<!--
+Add videos/screenshots of the user-facing interface in various display sizes (mainly phone and tablet) in both LTR and RTL mode to demonstrate that the changes made in this PR work correctly.
+[Oppia Android RTL Guide](https://github.com/oppia/oppia-android/wiki/RTL-Guidelines).
+-->
```

Comment:
Can we make this more actionable? E.g. maybe something like:

```
If your change includes PR changes, then:
- Add a video showing the full UX flow with a screen reader enabled (see accessibility guide)
- Add screenshots for portrait/landscape for both a tablet & phone of the before & after UI changes (see screenshot guide)
- For the screenshots above, include both English and pseudo-localized (RTL) screenshots (see RTL guide)
- Add a screenshot demonstrating that you ran affected Espresso tests locally & that they're passing
```

This might need to be moved/adjusted in later sections, but the key difference is to be as actionable as possible & to provide context since some of these steps may be difficult or confusing for people not familiar with how to do them.

Reply:
Hm, @BenHenning, you mean "UI changes" and not "PR changes", right?

Note -- Espresso tests are covered in the section below under UI. I'm wondering whether we should move some of this stuff down there too (to the deletable section) if it's all UI-related. Do we want contributors of non-UI PRs to add any proof of correctness?

Otherwise, I agree with making things as actionable as possible.

Reply:
I have tried to make changes as per suggestion. Have a look and let me know your thoughts.